index.html

```diff
@@ -3,12 +3,51 @@
 
 <head>
   <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no" />
   <title>🏎️ 자동차 경주 게임</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./src/css/index.css">
 </head>
 
 <body>
   <div id="app">
+    <div class="d-flex justify-content-center mt-5">
+      <div>
+        <section>
+          <h1 class="text-center">🏎️ 자동차 경주 게임🏁</h1>
+          <p>5자 이하의 자동차 이름을 콤마로 구분하여 입력해주세요.</p>
+        </section>
+        <section class="mb-5">
```

Comment:
```suggestion
        <section class="mb-5">
          <p>5자 이하의 자동차 이름을 콤마로 구분하여 입력해주세요.</p>
```
html 영역을 그룹핑할 때 그 안에 들어가는 요소들의 스타일이 동일한 경우 대체로 자식요소들이 동일한 위치에 있도록 하는게 좋아요.

마찬가지로 입력 필드에 대한 설명(또는 제목), 입력 필드, 버튼을 section 태그로 그룹핑했을 때, 자동차 이름 입력 필드 섹션과 시도 횟수 입력 필드 섹션에 자식 요소들이 동일하게 위치 하는게 더 일관성있는 코드가 되지 않을까요?

그리고 p태그보다 더 웹표준에 적합한, 입력 필드의 설명 또는 제목을 나타낼 수 있는 태그가 무엇일까요?

Reply:
1. 네, 이해했습니다. 수정하겠습니다.
2. p태그보다 label이 더 적합해보입니다.